app/components/emoji_picker/emoji_picker.js

```diff
@@ -33,171 +43,311 @@ class EmojiPicker extends PureComponent {
         onEmojiPress: emptyFunction
     };
 
-    leftButton = {
-        id: 'close-edit-post'
-    };
+    static contextTypes = {
+        intl: intlShape.isRequired
```

Comment:
Why not keep using injectIntl here instead of pulling this from the context?

Reply:
Figured why not since it's already available through context since the Root component wraps everything and makes it available without the need for additional imports or wrapping our export. 0/5 on keeping it or changing it. Just used it out of convenience.

Reply:
I really like this way better

Reply:
Oh yeah, that's fine by me if we're all in agreement. As you said, wrapping it is a pain, and it does a bunch of annoying things like making it so that you need to `getWrappedInstance()` if you're calling any methods.